specification/storage/resource-manager/readme.md

```diff
@@ -614,6 +614,31 @@ directive:
 
 ```
 
+### Tag: package-2018-03-preview-only
+
+These settings apply only when `--tag=package-2018-03-preview-only` is specified on the command line.
+
+``` yaml $(tag) == 'package-2018-03-preview-only'
+input-file:
+- Microsoft.Storage/preview/2018-03-01-preview/storage.json
+- Microsoft.Storage/preview/2018-03-01-preview/blob.json
+- Microsoft.Storage/preview/2018-03-01-preview/managementpolicy.json
+- 
```

Comment:
this seems wrong.

autorest can work on commit da0cfefaa0e6c237e1e3819f1cb2e11d7606878d, but not on latest main